generator/internal/genclient/parser/protobuf/protobuf.go

```diff
@@ -377,6 +365,34 @@ func normalizeTypes(state *genclient.APIState, in *descriptorpb.FieldDescriptorP
 
 }
 
+func processService(state *genclient.APIState, s *descriptorpb.ServiceDescriptorProto, sFQN, packagez string) *genclient.Service {
+	service := &genclient.Service{
+		Name:        s.GetName(),
+		ID:          sFQN,
+		Package:     packagez,
+		DefaultHost: parseDefaultHost(s.GetOptions()),
+	}
+	state.ServiceByID[service.ID] = service
+	return service
+}
+
+func processMethod(state *genclient.APIState, m *descriptorpb.MethodDescriptorProto, mFQN string) *genclient.Method {
+	pathInfo, err := parsePathInfo(m, state)
+	if err != nil {
+		slog.Error("unsupported http method", "method", m)
+		return nil
+	}
+	method := &genclient.Method{
+		ID:           mFQN,
+		PathInfo:     pathInfo,
+		Name:         m.GetName(),
+		InputTypeID:  m.GetInputType(),
+		OutputTypeID: m.GetOutputType(),
+	}
+	state.MethodByID[mFQN] = method
```

Comment:
I would expect an analogous change in `openapi.go` and `openapi_test.go`.

Reply:
done, openapi already had the ids in the test.